components/home/mcp-card.tsx

```
@@ -0,0 +1,185 @@
+            if (envVar && !(key === 'x-meeting-baas-api-key' || key === 'x-api-key')) {
+                obj[key] = envVar.value
+            }
+            inject(obj[key])
+        }
+    }
+    inject(config)
+    return config
+}
+
+// Recursively mask all sensitive values, including API key fields
+function maskSensitiveLocalConfig(localConfig: any, envVars: any[] = [], reveal: boolean = false, apiKey?: string) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)